src/Apps/W1/Shopify/App/src/Order Fulfillments/Tables/ShpfyFulfillmentLine.Table.al

```diff
@@ -58,6 +58,11 @@ table 30139 "Shpfy Fulfillment Line"
         {
             Clustered = true;
         }
-        key(Indx001; "Fulfillment Id", "Is Gift Card") { }
+        key(Indx001; "Fulfillment Id", "Is Gift Card")
```

Comment:
Lets use the normal naming on the keys, Key1,Key2,Key3....
This is introducing mess